src/ui/dom/ListView.ts

```diff
@@ -166,10 +188,29 @@ class ListView<T> extends SingleNodeWidget {
 			};
 		}
 
+		this._children = [];
 		this._widget.set('store', store);
 		this._collection = value;
 	}
 
+	destroy():void {
+		super.destroy();
+		ContainerMixin.prototype.destroy.call(this);
+	}
+
+	empty():void {
```

Comment:
It should not be possible to mutate the list of children in the listview except by passing in a filtered collection